src/main/java/org/sopt/app/application/stamp/StampService.java

```diff
@@ -23,130 +24,129 @@
 @RequiredArgsConstructor
 public class StampService {
 
-  private final StampRepository stampRepository;
+    private final StampRepository stampRepository;
 
-  private final UserRepository userRepository;
+    private final UserRepository userRepository;
 
-  private final MissionRepository missionRepository;
+    private final MissionRepository missionRepository;
 
-  public Stamp findStamp(String userId, Long missionId) {
-    return stampRepository.findByUserIdAndMissionId(Long.valueOf(userId), missionId);
-  }
+    @Transactional(readOnly = true)
+    public Stamp findStamp(String userId, Long missionId) {
+        return stampRepository.findByUserIdAndMissionId(Long.valueOf(userId), missionId);
+    }
 
-  @Transactional
-  public Stamp uploadStamp(StampRequestDto stampRequestDto, List<String> imgPaths, String userId,
-      Long missionId) {
-    List<String> imgList = new ArrayList<>(imgPaths);
-    Stamp stamp = this.convertStampImg(stampRequestDto, imgList, userId, missionId);
+    @Transactional
+    public Stamp uploadStamp(RegisterStampRequest stampRequest, List<String> imgPaths, String userId,
+            Long missionId) {
```

Comment:
```suggestion
    public Stamp uploadStamp(
        RegisterStampRequest stampRequest,
        List<String> imgPaths,
        String userId,
        Long missionId
    ) {
```
가독성 측면에서 파라미터 하나하나 집어주는 게 좋을 거 같은데,
그냥 suggestion입니다!